### ISSUE TITLE
egress not connected (redis required)

### ISSUE THREAD

**Issue by crapthings** [livekit/egress#344]
i'm trying to call record feature with egress, but i got an error

```

egressClient.startRoomCompositeEgress('TtzchQ8CTto2QiPts', {
      fileType:  EncodedFileType.MP4,
      filepath: '/Users/monsterstep/dev/meteor-playground/project-yzc/.uploads/room-composite-test.mp4',
    })
```

livekit egress docker logs

```
+ export TMPDIR=/tmp/lkegress
+ TMPDIR=/tmp/lkegress
+ rm -rf /var/run/pulse /var/lib/pulse /root/.config/pulse
+ pulseaudio -D --verbose --exit-idle-time=-1 --system --disallow-exit
W: [pulseaudio] main.c: Running in system mode, but --disallow-module-loading not set.
N: [pulseaudio] main.c: Running in system mode, forcibly disabling SHM mode.
I: [pulseaudio] main.c: Daemon startup successful.
+ '[' -z /tmp/lkegress ']'
+ mkdir -p /tmp/lkegress
+ exec egress
2023-04-21T12:10:46.674Z	INFO	egress	redis/redis.go:92	connecting to redis	{"nodeID": "NE_XAbTrArcjdDF", "clusterID": "", "simple": true, "addr": "192.168.50.105:10002"}
2023-04-21T12:10:46.676Z	DEBUG	egress	service/debug.go:24	debug handler disabled	{"nodeID": "NE_XAbTrArcjdDF", "clusterID": ""}
2023-04-21T12:10:46.676Z	DEBUG	egress	service/service.go:73	starting service	{"nodeID": "NE_XAbTrArcjdDF", "clusterID": "", "version": "1.6.3"}
2023-04-21T12:10:46.676Z	DEBUG	egress	service/templates_server.go:24	starting template server on address localhost:7980	{"nodeID": "NE_XAbTrArcjdDF", "clusterID": ""}
2023-04-21T12:10:46.676Z	INFO	egress	stats/monitor.go:133	available CPU cores: 16 max cost: 4.500000	{"nodeID": "NE_XAbTrArcjdDF", "clusterID": ""}
2023-04-21T12:10:46.676Z	DEBUG	egress	service/service_deprecated.go:27	service ready	{"nodeID": "NE_XAbTrArcjdDF", "clusterID": ""}
```

config.yaml

```
  log_level: debug
  api_key: devkey
  api_secret: secret
  ws_url: ws://192.168.50.105:7880
  insecure: true
  redis:
    address: 192.168.50.105:10002
```

```
docker run --rm \
    -e EGRESS_CONFIG_FILE=/out/config.yaml \
    -v /root/server/livekit-egress/out:/out \
    livekit/egress \
    livekit-egress




    docker exec -it redis redis-cli ping
```

**Comment by davidzhao** [comment]
Is livekit server connected to the same Egress instance?

**Comment by frostbyte73** [comment]
that error comes from your livekit-server config not having the redis connection

**Comment by crapthings** [comment]
> Is livekit server connected to the same Redis instance?

yes, if i provide the wrong redis host ip with config.yaml, the docker won't start at all

**Comment by crapthings** [comment]
> Is livekit server connected to the same Redis instance?

oh i think the prob is i don't specify redis with livekit-server

i just run this with bind 0.0.0.0

```
livekit-server --dev
```

**Comment by crapthings** [comment]
it works after specify the redis port

```
pm2 start --name livekit-server livekit-server -- --dev --bind 0.0.0.0 --redis-host 192.168.50.105:10002
```


```
2023-04-22T05:22:24.611Z	INFO	egress	redis/redis.go:92	connecting to redis	{"nodeID": "NE_krs4D7TZp9Ns", "clusterID": "", "simple": true, "addr": "192.168.50.105:10002"}
2023-04-22T05:22:24.613Z	DEBUG	egress	service/debug.go:24	debug handler disabled	{"nodeID": "NE_krs4D7TZp9Ns", "clusterID": ""}
2023-04-22T05:22:24.613Z	DEBUG	egress	service/service.go:73	starting service	{"nodeID": "NE_krs4D7TZp9Ns", "clusterID": "", "version": "1.6.3"}
2023-04-22T05:22:24.613Z	DEBUG	egress	service/templates_server.go:24	starting template server on address localhost:7980	{"nodeID": "NE_krs4D7TZp9Ns", "clusterID": ""}
2023-04-22T05:22:24.613Z	INFO	egress	stats/monitor.go:133	available CPU cores: 16 max cost: 4.500000	{"nodeID": "NE_krs4D7TZp9Ns", "clusterID": ""}
2023-04-22T05:22:24.613Z	DEBUG	egress	service/service_deprecated.go:27	service ready	{"nodeID": "NE_krs4D7TZp9Ns", "clusterID": ""}
2023-04-22T05:22:29.518Z	WARN	egress	service/service_deprecated.go:41	Using deprecated egress client. Upgrade livekit-server to v1.3.5+ and add egress:use_psrpc:true to your livekit config	{"nodeID": "NE_krs4D7TZp9Ns", "clusterID": ""}
2023-04-22T05:22:29.519Z	INFO	egress	service/service_deprecated.go:117	request accepted	{"nodeID": "NE_krs4D7TZp9Ns", "clusterID": "", "egressID": "EG_H97TVbd8DTJT", "requestID": "RPC_U4aEwpJtbdkd", "senderID": "ND_fgoEy733"}
2023-04-22T05:22:29.530Z	DEBUG	egress	server/main.go:169	handler launched	{"nodeID": "NE_krs4D7TZp9Ns", "handlerID": "EGH_bWJETejXzDwc", "clusterID": "", "egressID": "EG_H97TVbd8DTJT"}
2023-04-22T05:22:29.530Z	INFO	egress	redis/redis.go:92	connecting to redis	{"nodeID": "NE_krs4D7TZp9Ns", "handlerID": "EGH_bWJETejXzDwc", "clusterID": "", "egressID": "EG_H97TVbd8DTJT", "simple": true, "addr": "192.168.50.105:10002"}
2023-04-22T05:22:29.533Z	DEBUG	egress	source/web.go:137	launching xvfb	{"nodeID": "NE_krs4D7TZp9Ns", "handlerID": "EGH_bWJETejXzDwc", "clusterID": "", "egressID": "EG_H97TVbd8DTJT", "display": ":483377136", "dims": "1920x1080x24"}
2023-04-22T05:22:29.533Z	DEBUG	egress	source/web.go:172	launching chrome	{"nodeID": "NE_krs4D7TZp9Ns", "handlerID": "EGH_bWJETejXzDwc", "clusterID": "", "egressID": "EG_H97TVbd8DTJT", "url": "http://localhost:7980/?layout=&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2ODIyMjczNDksImlzcyI6ImRldmtleSIsIm5iZiI6MTY4MjE0MDk0OSwic3ViIjoiRUdfSDk3VFZiZDhEVEpUIiwidmlkZW8iOnsiY2FuUHVibGlzaCI6ZmFsc2UsImNhblB1Ymxpc2hEYXRhIjpmYWxzZSwiY2FuU3Vic2NyaWJlIjp0cnVlLCJoaWRkZW4iOnRydWUsInJlY29yZGVyIjp0cnVlLCJyb29tIjoiVHR6Y2hROENUdG8yUWlQdHMiLCJyb29tSm9pbiI6dHJ1ZX19.vjU-p5QEDciQh8sexMdnfBxZHzh_wR1zfTuKHBH9eUc&url=ws%3A%2F%2F192.168.50.105%3A7880"}
2023-04-22T05:22:30.093Z	INFO	egress	source/web.go:240	chrome: START_RECORDING	{"nodeID": "NE_krs4D7TZp9Ns", "handlerID": "EGH_bWJETejXzDwc", "clusterID": "", "egressID": "EG_H97TVbd8DTJT"}
2023-04-22T05:22:32.620Z	INFO	egress	pipeline/watch.go:152	pipeline playing	{"nodeID": "NE_krs4D7TZp9Ns", "handlerID": "EGH_bWJETejXzDwc", "clusterID": "", "egressID": "EG_H97TVbd8DTJT"}
2023-04-22T05:22:32.621Z	INFO	egress	service/handler_deprecated.go:157	egress updated	{"nodeID": "NE_krs4D7TZp9Ns", "handlerID": "EGH_bWJETejXzDwc", "clusterID": "", "egressID": "EG_H97TVbd8DTJT", "egressID": "EG_H97TVbd8DTJT", "request_type": "room_composite", "output_type": "file", "status": "EGRESS_ACTIVE"}
```

**Comment by crapthings** [comment]
i've tried configurate an cloud storage, but still no file in bucket
